### PR TITLE
Update torch.fx docs

### PR DESCRIPTION
**Pull request by msaroufim**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: torch/fx/README.md [hunk 1]
```diff
@@ -41,7 +39,7 @@ By using `nn.Module` as the interface between passes, FX transforms are interope
 
 The following sections will walk us through the components that transform from original `torch.nn.Module` to FX IR and finally to generated Python code and a GraphModule instance:
 
-FX’s front-end makes use of the dynamic nature of Python to intercept call-sites for various entities (PyTorch operators, Module invocations, and Tensor method invocations). This functionality is exposed through an API called `torch.fx.symbolic_trace`.  We can see how this works by way of an example:
+FX’s front-end makes use of the dynamic nature of Python to intercept call-sites for various entities (PyTorch operators, Module invocations, and Tensor method invocations). The simplest way to get an fx graph is via `torch.fx.symbolic_trace`.  We can see how this works by way of an example:
```
Comment (svekars) [inline torch/fx/README.md:42]:
```suggestion
FX’s front-end makes use of the dynamic nature of Python to intercept call-sites for various entities (PyTorch operators, Module invocations, and Tensor method invocations). The simplest way to get an FX graph is by using `torch.fx.symbolic_trace`.  We can see how this works by way of an example:
```

**Review by svekars — APPROVED** [review]
One tiny nit, but otherwise LGTM.

**Comment by msaroufim** [comment]
@SherlockNoMad looks like we need your approval as well, do you mind stamping plz?

File: torch/fx/README.md [hunk 2]
```diff
@@ -132,3 +140,6 @@ def forward(self, x):
 ```
 
 This is the core of why FX is a Python-to-Python translation toolkit. Outside users can treat the results of FX transformations as they would any other `nn.Module` instance.
+
+# Next steps
+If you're interested in learning more about obtaining fx graphs, which kinds of IRs are available to you and how to execute simple transformations make sure to check out [this tutorial](https://colab.research.google.com/drive/1Zh-Uo3TcTH8yYJF-LLo5rjlHVMtqvMdf)
```
Comment (SherlockNoMad) [inline torch/fx/README.md:145]: lol... thanks for promoting this.... 
I am on the hook of keeping this up-to-date....

**Review by SherlockNoMad — APPROVED** [review]

**Pull request closed without merging**